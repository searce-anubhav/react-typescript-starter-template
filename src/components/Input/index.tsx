import * as React from 'react';

interface InputProps {
	className: string;
	placeholder: string;
	type: string;
}

const Input: React.FC<InputProps> = ({
	className,
	placeholder,
	type,
}): JSX.Element => {
	return (
		<React.Fragment>
			<input className={className} type={type} placeholder={placeholder}></input>
		</React.Fragment>
	);
};

export default Input;
