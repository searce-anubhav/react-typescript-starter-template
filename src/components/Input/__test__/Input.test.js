import React from 'react';
import renderer from 'react-test-renderer';
import Input from 'components/Input';

test('Render Input Snapshot and Match', () => {
	const component = renderer.create(<Input />);
	let tree = component.toJSON();
	expect(tree).toMatchSnapshot();
});
