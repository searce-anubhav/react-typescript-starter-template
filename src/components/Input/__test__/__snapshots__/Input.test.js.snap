// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Input Snapshot and Match 1`] = `<input />`;
